webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -101,42 +84,6 @@ export default class SuggestionList extends React.PureComponent<Props> {
         }
     };
 
-    announceLabel() {
```

Comment:
This is from the old way of using `aria-live` to read out the selected autocomplete result. This isn't needed any more since we're using `aria-activedescendant`